examples/CMakeLists.txt

```diff
@@ -53,6 +53,13 @@ if(GINKGO_BUILD_MPI)
     list(APPEND EXAMPLES_LIST distributed-solver)
 endif()
 
+find_package(Kokkos QUIET)
+if(Kokkos_FOUND)
+    list(APPEND EXAMPLES_LIST kokkos_assembly)
```

Comment:
Could you add a check for the ginkgo CCache setting, and print a warning (or disable it?) in that case?

Reply:
I would print a warning and then leave it to the user to disable that.